### PR TITLE
[DEVHUB-1111] Added GTM to NextJS

### PR DESCRIPTION
**Pull request by gfunk1230**
Added GTM to NextJS App.

### CODE REVIEW

File: src/pages/_document.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import Document, { Html, Head, Main, NextScript } from 'next/document';
```
Comment (ajhenri) [inline src/pages/_document.tsx:1]: I did not know about next/document. This can be used to alter what's in the body tag. Neat!

**Review by ajhenri — APPROVED** [review]
LGTM

File: src/pages/_app.tsx [hunk 2]
```diff
@@ -1,18 +1,46 @@
...
 import theme from '@mdb/flora/theme';
 import { ThemeProvider } from '@theme-ui/core';
-
+import { GTM_ID, pageView } from '../lib/gtm';
 import Layout from '../components/layout';
 
 function MyApp({ Component, pageProps }: AppProps) {
+    {
+        /* Is used to track route changes and send page views to Google Tag Manager */
+    }
```
Comment (ajhenri) [inline src/pages/_app.tsx:14]: @gfunk1230 I approved but wanted to point out that the brackets aren't needed here.

Comment (gfunk1230) [reply src/pages/_app.tsx:14]: Good catch! will remove!

**Review by mcolella14 — APPROVED** [review]

**Pull request merged**